arccore/src/message_passing_mpi/arccore/message_passing_mpi/MpiDatatype.h

```diff
@@ -101,10 +104,13 @@ template<typename RealType> inline
 void ARCCORE_MPIOP_CALL StdMpiReduceOperator<RealType>::
 _MaxOperator(void* a,void* b,int* len,MPI_Datatype* type)
 {
-  ARCCORE_UNUSED(type);
+  Integer nb_elem_in_realtype = 0;
+  MPI_Type_size(*type, &nb_elem_in_realtype);
+  nb_elem_in_realtype /= sizeof(RealType);
```

Comment:
Is this always true? Can padding change the size and possibly the guess of the number of elements.